src/entries/popup/hooks/useHomeShortcuts.ts

```diff
@@ -246,3 +246,8 @@ export function useHomeShortcuts() {
     handler: handleHomeShortcuts,
   });
 }
+
+export function HomeShortcuts() {
+  useHomeShortcuts();
+  return null;
+}
```

Comment:
nice, i saw this in your tx animation pr and was gonna ask about it 
👌

Reply:
This is really nice and creative! Now we can have `<HomeShortcuts />` re-rendering and not whole `<Home />` component 🚀